doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -2032,36 +2078,39 @@ Response:
         [
           {
             "id": "abcdefg",
-            "subject": {
+            "description": "Example Context Source",
+            "dataProvided": {
               "entities": [
                 {
                   "id": "Bcn_Welt",
                   "type": "Room"
                 }
               ],
-              "attributes": [
+              "attrs": [
                 "temperature"
               ]
             },
-            "callback": "http://weather.example.com/ngsi",
-            "metadata": {
-              "providingService": {
-                 "value": "weather.example.com",
-                 "type": "Text"
-               },
-              "providingAuthority": {
-                 "value": "AEMET - Spain",
-                 "type": "Text"
-               }
+            "provider": {
+              "http": {
+                "url": "http://contextsource.example.org/v2/entities"
+              },
+              "legacy": false,
+              "supportedForwardingMode": "all"
             },
-            "duration": "PT1M"
+            "expires": "2017-10-31T12:00:00",
+            "status": "active",
+            "forwardingInformation": {
+              "timesSent": 12,
+              "lastForwarding": "2017-10-05T16:00:00.00Z",
+              "lastFailure": "2017-10-06T16:00:00.00Z"
```

Comment:
`"status": "active"` and `lastFailure` equal to `lastForwarding` is an impossible combination :)

Simple solution: change `lastFailure` to `lastSuccess`. Not only here, but in all the examples with the same problem.

Reply:
fixed 78ce71dfa1b2c6952018ede3a9ef640c65abc1dd